### ISSUE TITLE
Spammed with a lot of folders titled 'X11' inside `bin` when wanting to add DMD compiler on project creation

### ISSUE THREAD

**Issue by therealbluepandabear** [intellij-dlanguage/intellij-dlanguage#775]
Hello,

When creating a new project with your plugin and pressing the following:

![image](https://user-images.githubusercontent.com/50536495/201259613-6ef0d8cc-0a01-42a0-a5e3-f9c8360ef759.png)

And then navigating to `bin` directory, I get spammed with the following folders for no reason (these folders don't exist):

![image](https://user-images.githubusercontent.com/50536495/201259724-eaa591af-7fcb-490c-83d6-4c00d25da4ce.png)

This bug was caught on my PC which was running Linux.

Hopefully you guys can reproduce this bug, as I have tried to also uninstall and reinstall the plugin to validate that it's not a problem on my side, which it doesn't seem to be.

**Comment by SingingBush** [comment]
That's really odd. I use Linux (Fedora) almost exclusively and haven't seen that. What DMD version is it and how was it installed?

**Comment by therealbluepandabear** [comment]
Latest DMD installed via the official website.  I will try to get more details tomorrow. I

**Comment by etienne02** [comment]
Can you check that `X11` symbolic link does not point to `bin` creating an infinite loop of symlink?

**Comment by therealbluepandabear** [comment]
> Can you check that `X11` symbolic link does not point to `bin` creating an infinite loop of symlink?

I don't know how to check that, sorry.

**Comment by SingingBush** [comment]
@therealbluepandabear the screenshot you posted the other day doesn't show the location of that bin folder but I guess it's not the dmd compiler. I just tried to replicate this with the _dmd.2.100.2.linux.tar.xz_ download and found no issues with the 32 bit or 64 bit paths.

![image](https://user-images.githubusercontent.com/870567/201469401-40dcda3a-ff49-4c44-8471-f28ad9fc8041.png)

I also looked in my `/bin` and `/usr/bin` and didn't get that problem. I did a quick google search and found: https://askubuntu.com/questions/191654/why-are-there-infinitely-many-x11-subdirectories-in-usr-bin-x11

It seems that Ubuntu put that symlink there for compatibility reasons: https://askubuntu.com/a/191658

So it's a problem with Ubuntu.
